mifosng-android/src/main/java/com/mifos/mifosxdroid/online/generatecollectionsheet/GenerateCollectionSheetPresenter.java

```diff
@@ -83,44 +118,65 @@ public void onCompleted() {
 
                     @Override
                     public void onError(Throwable e) {
-                        getMvpView().showFetchingError("Failed to fetch Staff in Office");
+                        getMvpView().showProgressbar(false);
+                        try {
```

Comment:
use this utils method to show error https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/java/com/mifos/utils/MFErrorParser.java#L25